tests/SampleData.test.ts

```diff
@@ -1,3 +1,15 @@
+/*******************************************************************************
+
+    The sample block data for the test.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+ *******************************************************************************/
```

Comment:
This part is the same as the committed contents of bf254477

Reply:
Oh, it's my mistake. I fixed it.